anvil/src/winit.rs

```diff
@@ -361,7 +361,9 @@ pub fn run_winit() {
             match render_res {
                 Ok((damage, states)) => {
                     let has_rendered = damage.is_some();
-                    if let Some(damage) = damage {
+                    if let Some((sync, damage)) = damage {
+                        // wait for the rendering to finish
+                        sync.wait();
```

Comment:
I wonder if this will not just cause additional delay (as a proper compositor would block on the submitted dmabuf anyway). For true explicit sync we would send the fence to the compositor, but I think we can continue to rely on implicit sync here and just drop the sync point?

Reply:
removed the explicit wait